src/js/common/components/RequiredLoginView.jsx

```diff
@@ -113,7 +137,33 @@ class RequiredLoginView extends React.Component {
       }
     } else if (this.props.authRequired === 3) {
       if (this.state.accountType === 3) {
-        view = this.props.children;
+        if (_.isNil(this.state.profileStatus)) {
+          // If va_profile is null, the system is down and we will show system down message.
+          view = <SystemDownView messageLine1="Sorry, our system is temporarily down while we fix a few things. Please try again later."/>;
+        } else if (_.isEqual(this.state.profileStatus, 'not found')) {
+          // If va_profile is "not found", we cannot find you in our system and we will show a, we can't find you message.
+          view = <SystemDownView messageLine1="Sorry, our system was not able to match your record. For more information please call 1-800-XXX-XXXX."/>;
```

Comment:
Recommending:
"We couldn't find your records with that information. Please call support at 1-855-574-7286."
Julie made a good point about treading carefully with this kind of error to avoid sounding like we're saying they're not a Veteran.